.github/workflows/test-pr.yml

```diff
@@ -1,7 +1,6 @@
 name: 'Test PR'
 on:
   pull_request:
-    types: [opened, edited, reopened]
```

Comment:
Instead of removing add, this was a miss on my part in the last push. 
types: [opened, edited, reopened, **synchronize**]

Reply:
Done.